cmd/template/cluster/provider/templates/aws/bastion_aws_machine_template.yaml.tmpl

```diff
@@ -0,0 +1,36 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
```

Comment:
A lot of these aws types should be cleaned up to be much more minimal and let the webhook do more work. I followed the current output of `kubectl-gs` to create these outputs for now.